examples/vllm-qwen3-run.yaml

```
@@ -0,0 +1,108 @@
+# Example llama-stack configuration for Alibaba Qwen3 using vLLM (no RAG)
+
+# 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)